package.json

```diff
@@ -180,6 +180,7 @@
 		"circular-dependency-plugin": "5.2.0",
 		"concurrently": "5.0.1",
 		"cross-env": "6.0.3",
+		"duplicate-package-checker-webpack-plugin": "3.0.0",
```

Comment:
This should cause a `package-lock.json` change, too.

Reply:
I installed with `npm i -D …` and package-lock wasn't updated. I'd speculate it may have been there as hoisted transitive dep from devDep calypso-build? Please try to update the lock and confirm 👍

Reply:
Interesting, regenerating `package-lock.json` doesn't show any relevant changes. The `depenencies` array is duplicated in the `package-lock.json`, so I expected at least one new line 🤷‍♂

Reply:
> Interesting, regenerating `package-lock.json` doesn't show any relevant changes. The `depenencies` array is duplicated in the `package-lock.json`, so I expected at least one new line 🤷‍♂

We came across something similar in https://github.com/Automattic/wp-calypso/pull/38721#discussion_r364742817 as well. There may be some nuance to package-lock I haven't understood yet 😕